source/packages/libraries/core/deployment-helper/src/customResources/appConfigOverride.customResource.ts

```diff
@@ -16,6 +16,7 @@ import { TYPES } from '../di/types';
 import { logger } from '../utils/logger';
 import {CustomResourceEvent} from './customResource.model';
 import { CustomResource } from './customResource';
+import AWS from 'aws-sdk'
```

Comment:
annoying these customResource modules were implicitly importing the AWS namespace somehow, which was causing problems using the bundled `aws-sdk`, so no we're explicitly importing it